content/docs/tasks/telemetry/gateways/index.md

```diff
@@ -0,0 +1,632 @@
+---
+title: Remotely Accessing Telemetry Add-ons
+description: This task shows you how to configure external access to the set of Istio telemetry add-ons.
+weight: 99
+keywords: [telemetry,gateway,jaeger,zipkin,tracing,kiali,prometheus,add-ons]
+---
+
+This task shows how to configure Istio to expose and access the telemetry add-ons outside of
+a cluster.
+
+## Configuring Remote Access
+
+Access to the telemetry add-ons can be configured in a number of different ways. This task will cover
+two basic access methods: secure (via https) and insecure (via http). The secure method is *strongly
+recommended* for any production or sensitive environment. Insecure access is simpler to set up, but
+will not protect any credentials or data transmitted outside of your cluster.
+
+### Option 1: Secure Access (HTTPS)
+
+A server certificate is required for secure access. Follow these steps to install and configure
+server certificates for a domain that you control.
+
+It is possible to use self-signed certificates instead, but that is not covered by this task.
+[Securing Gateways with HTTPS Using Secret Discovery Service](/docs/tasks/traffic-management/secure-ingress/sds/)
+provides general details on using self-signed certificates to access in-cluster services.
+
+{{< warning >}}
+This task is concerned *solely* with securing the transport layer for access. When exposing the telemetry
+services externally, the services should be installed and configured to require authentication.
+{{< /warning >}}
+
+1. [Install Istio](/docs/setup/) in your cluster with `cert-manager` enabled and `istio-ingressgateway` configured to use
```

Comment:
Based on the commands you use, I'm assuming you need people to install Istio on a K8s cluster. Please be more specific on the documents you link. I've suggested a change to point to the K8s setup landing page but if there is a specific installation flow you want readers to follow, feel free to change the link accordingly.

```suggestion
1. [Install Istio](/docs/setup/kubernetes/) in your cluster and enable the `cert-manager` flag and configure the `istio-ingressgateway` to use
```